man/fi_info.1.md

```diff
@@ -0,0 +1,109 @@
+---
+layout: page
+title: fi_info(1)
+tagline: Libfabric Programmer's Manual
+---
+{% include JB/setup %}
+
+
+# NAME
+
+fi_info  \- Simple utility to query for fabric interfaces
+
+
+# SYNOPSYS
+```
+ fi_info [OPTIONS]
+```
+
+# DESCRIPTION
+
+The fi_info utility can be used to query for available fabric interfaces. The
+utility supports filtering based on a number of options such as endpoint type,
+provider name, or supported modes. Additionally, fi_info can also be used to
+discover the environment variables that can be used to tune provider specific
+parameters.
+
+# OPTIONS
+
+## Filtering
+
+*-n, --node=\<NAME\>*
+: Node name or address used to filter interfaces. Only interfaces which can
+reach the given node or address will respond.
+
+*-p, --port=\<PORT\>*
+: Port number used to filter interfaces.
+
+*-c, --caps=\<CAP1|CAP2\>..*
+: Pipe separated list of capabilities used to filter interfaces. Only
+interfaces supporting all of the given capabilities will respond. For more
+information on capabilities, see fi_getinfo(3).
+
+*-m, --mode=\<MOD1|MOD2\>..*
+: Pipe separated list of modes used to filter interfaces. Only interfaces
+supporting all of the given modes will respond. For more information on, modes
+see fi_getinfo(3).
+
+*-t, --ep_type=\<EPTYPE\>*
+: Specifies the type of fabric interface communication desired. For example,
+specifying FI_EP_DGRAM would return only interfaces which support unreliable
+datagram. For more information on endpoint types, see fi_endpoint(3).
+
+*-a, --addr_format=\<FMT\>*
+: Filter fabric interfaces by their address format. For example,
+specifying FI_SOCKADDR_IN would return only interfaces which use sockaddr_in
+structures for addressing. For more information on address formats, see
+fi_getinfo(3).
+
+*-f, --provider=\<PROV\>*
+: Filter fabric interfaces by the provider implementation. For a list of
+providers, see the `--list` option.
+
+## Discovery
+
+*-e, --env*
+: List libfabric related environment levels which can be used to enable extra
+configuration or tuning.
+
+*-l, --list*
+: List available libfabric providers.
+
+*-v, --verbose*
+: By default, fi_info will display a summary of each of the interfaces
+discovered. If the verbose option is enabled, then all of the contents of the
+fi_info structure are displayed. For more information on the data contained in
+the fi_info structure, see fi_getinfo(3).
+
+*--version*
+: Display versioning information.
+
+# USAGE EXAMPLES
+
+```
+$ fi_info 30.0.11.1 -f usnic -t FI_EP_DGRAM
```

Comment:
> "-n" is missing before "30.0.11.1" here. This example won't work as explained, all the arguments are ignored.

Oops, yeah both of these were typos. I copied the output but not the command. Fixed.